misc/ubuntu-touch/manifest.json.in

```diff
@@ -1,6 +1,6 @@
 {
     "name": "im.kaidan.kaidan",
-    "description": "Simple and user-friendly Jabber/XMPP client for every device ",
+    "description": "A Simple and user-friendly Jabber/XMPP client for every device",
```

Comment:
Since this is a sentence and not a title, simple should be written with a normal letter, not a capital one.